tests/virt_autotest/virtual_network_utils.pm

```diff
@@ -0,0 +1,136 @@
+# SUSE's openQA tests
+#
+# Copyright (C) 2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+package virtual_network_utils;
```

Comment:
This should be moved to `lib/`

Reply:
Good question. I had think this too. For now, it just only worked for the libvirt virtual network testing. So, at last, I had saved it under tests/virt_autotest dir.

Reply:
No @guoxuguang, this is not a question. In my opinion this strongly belongs under `lib/` directory. There is no problem putting this as `lib/virt_autotest/virtual_network_utils.pm ` for example.

Reply:
@alice-suse would we move virtual_network_utils.pm to `lib/` ?

Reply:
@pdostal had moved virtual_network_utils.pm into lib/virt_autotest

Reply:
Had updated code changed